src/main/java/uk/gov/hmcts/reform/unspec/enums/ResponseType.java

```diff
@@ -1,6 +1,6 @@
 package uk.gov.hmcts.reform.unspec.enums;
 
-public enum DefendantResponseType {
+public enum ResponseType {
```

Comment:
should we rename this to `RespondentResponseType` to distinguish with `ApplicantResponseType` ?

Reply:
done